mass_mailing_list_dynamic/wizards/mail_mass_mailing_load_filter.py

```diff
@@ -12,7 +12,9 @@ class MassMailingLoadFilter(models.TransientModel):
         comodel_name='ir.filters',
         string="Filter to load",
         required=True,
-        domain=[('model_id', '=', 'res.partner')],
+        domain="[('model_id', '=', 'res.partner'), '|', "
+               "('user_id', '=', uid), ('user_id','=',False)]",
```

Comment:
This patch is good, although I'd prefer it if you comment something in the commit message about it.